bin/cml/send-comment.js

```diff
@@ -27,6 +25,25 @@ exports.builder = (yargs) =>
         default: 'HEAD',
         description: 'Commit SHA linked to this comment'
       },
+      publish: {
+        type: 'boolean',
+        description:
+          'Upload local files and images linked from the Markdown report'
+      },
+      watch: {
+        type: 'boolean',
```

Comment:
since `--update=true` is forced by `--watch`, it doesn't make sense to accept an `--update` flag, right?

```suggestion
        type: 'boolean',
        conflicts: ['update'],
```

Reply:
Yes, it does; see https://github.com/iterative/cml/pull/1026#discussion_r894300912

Reply:
The `--update` flag controls whether to:
- create a new comment and update it on every watcher event (unset), or
- use an existing comment and update it on every watcher event (set)

Reply:
Oh, completely missed this. Really? Seems very complex, overloading `--update` to mean two different things.

- `cml send-comment`: comment
- `cml send-comment --update`: update comment
- `cml send-comment --watch`: comment & periodically update
- `cml send-comment --watch --update`: update comment & periodically update

:nauseated_face:

Can we address #762 first before this PR?

`cml report [--pr] {create [--rm-watermark],update,watch [--update]}` seems 10x more natural and easier to explain/justify.

Reply:
resolving unresolved